source/404.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+permalink: /404.html
+title: 404 Page Not Found
+---
+
+# 404 Page Not Found
+
+The page was moved or never existed.  
+Use menu above to navigate.  
+You will be redirected to main page in 5 second.
+
+<div onLoad="setTimeout('redirectToHome()', 5000)">
```

Comment:
I think only `body` has `onLoad` attribute. I think below instead would work:

```diff
-<div onLoad="setTimeout('redirectToHome()', 5000)">
 <script type="text/javascript">
    function redirectToHome() {
        window.location = "http://struts.apache.org/"
    }
+  setTimeout(redirectToHome, 5000);
```

Reply:
Thanks.

First, it works nice! See http://struts.apache.org/missing-page

changes as #76